test/integration/ext/openai/model.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+import vcr
+
+from superduperdb.ext.openai.model import OpenAIChatCompletion, OpenAIEmbedding
+
+
+@vcr.use_cassette(
+    'test/integration/ext/openai/cassettes/test_embed.yaml',
+    filter_headers=['authorization'],
+)
+def test_embed():
+    e = OpenAIEmbedding(model='text-embedding-ada-002')
+    resp = e.predict('Hello, world!')
```

Comment:
These are very neat tests. For the first time in this project, I might even suggest losing an empty line here and there!